src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -769,7 +769,7 @@ namespace
         {
             const int DefaultMaxBounces = 8;
 
-            const int max_bounces = get_config<int>(config, param_path, -1);
+            const int max_bounces = get_config<int>(config, param_path, DefaultMaxBounces);
```

Comment:
To clarify things further, I would suggest to:

- Rename `load_bounce_settings()` to `load_global_max_bounce_settings()`.
- Pass the default value in argument to the function.

What do you think?

Reply:
Yes I agree. I think that way it is also more clear what default value is being set in load_separate_bounce_settings as well. I'll make the necessary changes and commit again.

Reply:
The only issue is that load_bounce_settings is used  for photon mapping as well. So, I am going to change it to load_max_bounce_settings as global won't be applicable for photon mapping.